docs/how-to/backup-your-databases.md

```diff
@@ -338,6 +338,29 @@ within the database dump:
 borgmatic restore --archive latest --database users --schema tentant1
 ```
 
+### Restore to an alternate host
+<span class="minilink minilink-addedin">New in version 1.7.15</span> You
+can restore a database dump to a host other than the one from which it was
+originally dumped. You can also change other connection parameters like the
+username, password, and port. This can be done from the command line:
```

Comment:
Nit: The info here is good, but just from an editing prose perspective, there's lots of "you can" and "can be done" going on in this paragraph. A little of that is totally fine of course; I'm just calling out the repeated use.